packages/labs/ssr/src/test/integration/client/setup.ts

```diff
@@ -169,164 +172,170 @@ const assertHTML = (
   }
 };
 
-suite('basic', () => {
-  let container: HTMLElement;
-  let observer: MutationObserver;
-  let _mutations: Array<MutationRecord>;
+const modes = ['sandboxed', 'global'] as const;
+export const setupTest = async (
+  tests: SSRTestSuite,
+  testFile: string,
+  mode: typeof modes[number] = 'sandboxed'
+) => {
```

Comment:
The diff here is horrible but the main change is just sticking the whole suite creation in a parameterized function and calling it from the actual `*_test.js` files. The `mode` is used to switch which server endpoint is hit (global vs. sandbox) on L247.